File: RB101_Programming_Foundations/small_problems/Easy2/Run1/odd_numbers.rb
# Print all odd numbers from 1 to 99, inclusive. All numbers should be printed on separate lines.

count = 1

loop do
  puts count
  count += 2
  break if count == 99
end

# (1..100).each {|num| puts num if num.odd?}

# count.upto(99) {|num| puts num if num.odd?}

File: js210_fundamentals_of_JS_Programming/problem_sets/easy1/odd_numbers.js
// Log all odd numbers from 1 to 99, inclusive, to the console. Log all numbers on separate lines.

const readlineSync = require('readline-sync');

limit = readlineSync.question('Print odd numbers up to what number? ');

for (let i = 1; i <= limit; i += 2) {
  console.log(i);
}
